src/popup/components/TransactionList.vue

```diff
@@ -71,12 +71,15 @@ import {
   useTransactionAndTokenFilter,
   useTransactionTx,
 } from '../../composables';
+import { createPollingBasedOnMountedComponents } from '../../composables/composablesHelpers';
```

Comment:
In my opinion this is not a good change to use this composable helper inside of a component. The reason for this helper was to be used inside of the composables - functions used within many components. This is why it has the mounted components counter.